### ISSUE TITLE
Ports

### ISSUE THREAD

**Issue by Javrd** [ramkulkarni1/django-apache2-docker#1]
First and foremost, thanks to share this example about how deploy django wsgi with apache on docker. 

I just wanted to make note of ports configuration. At readme.md you say app can be access by port 8500, that is not true. As we can see at docker-compose, accessible ports are 8005, 3500 and 8006. In the other hand, port 8006 is pointing container port 81, that is not exposed on dockerfile. Finally, port 3500 goes to exposed port 3500, but I think there is nothing listening to this port. 

So, If I am right, you only need to expose port 80 on your dockerfile and configure port 8500 on your docker-compsoe to point it, or change readme.md so access link has port 8005 instead of port 8500.

**Comment by fermenreq** [comment]
+1 @Javrd

**Comment by jingbi72** [comment]
Hi, I tried this docker, and have one issue.
When docker is up, I can connect server on local machine, But when I try to connect from externel network, It return 500 error.

**Comment by AlessandroTaufer** [comment]
True, have a look [at this fork](https://github.com/AlessandroTaufer/django-apache2-docker) if you need a working container.
